services/movie.py

```diff
@@ -28,13 +33,15 @@
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Similar to the `get_movies` function, consider using an empty list as a default value for `genres_ids` and `actors_ids` to avoid potential issues with `NoneType` operations. Ensure proper handling to avoid mutable default argument pitfalls.